src/components/extractor.ts

```diff
@@ -98,21 +98,26 @@ export function extractProps(propsSymbol: ts.Symbol, checker: ts.TypeChecker) {
     .sort((a, b) => a.name.localeCompare(b.name));
 }
 
-export function extractFunctions(propsSymbol: ts.Symbol, checker: ts.TypeChecker) {
-  const propsName = propsSymbol.getName();
+export function extractTypes(propsSymbol: ts.Symbol, checker: ts.TypeChecker) {
   const namespaceDeclaration = [
     // if we got the namespace directly
     ...(propsSymbol.getDeclarations() ?? []),
     // find namespace declaration from the interface with the same name
     ...(checker.getDeclaredTypeOfSymbol(propsSymbol).getSymbol()?.getDeclarations() ?? []),
   ].find(decl => decl.kind === ts.SyntaxKind.ModuleDeclaration);
-  const refType = unwrapNamespaceDeclaration(namespaceDeclaration)
-    .map(child => checker.getTypeAtLocation(child))
-    .find(type => (type.getSymbol() ?? type.aliasSymbol)?.getName() === 'Ref');
+  return unwrapNamespaceDeclaration(namespaceDeclaration).map(node => {
+    if (!ts.isTypeAliasDeclaration(node) && !ts.isInterfaceDeclaration(node)) {
+      throw new Error(`Unexpected node type ${ts.SyntaxKind[node.kind]}`);
+    }
```

Comment:
This (and many other throws in this project) only exist to satisfy typescript type. That's why no coverage, because it is not a real use-case.